docs/yul.rst

```diff
@@ -15,13 +15,16 @@ It can already be used for "inline assembly" inside Solidity and
 future versions of the Solidity compiler will even use Yul as intermediate
 language. It should also be easy to build high-level optimizer stages for Yul.
 
-.. note::
+.. warning::
 
-    Note that the flavour used for "inline assembly" does not have types
+    Yul does not yet support types
```

Comment:
The thing is that it is still a different language. So "strict assembly" is implemented (and that's also what it is called when using the commandline interface), but "Yul" as described below is not.